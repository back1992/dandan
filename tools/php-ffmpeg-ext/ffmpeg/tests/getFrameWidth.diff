001+ Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/getFrameWidth.php on line 2
001- ffmpeg getFrameWidth(): 190