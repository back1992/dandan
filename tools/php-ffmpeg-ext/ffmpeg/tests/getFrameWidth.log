
---- EXPECTED OUTPUT
ffmpeg getFrameWidth(): 190
---- ACTUAL OUTPUT
Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/getFrameWidth.php on line 2
---- FAILED
