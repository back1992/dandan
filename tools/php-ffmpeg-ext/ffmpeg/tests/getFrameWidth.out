Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/getFrameWidth.php on line 2